codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2ConnectionEncoder.java

```diff
@@ -384,7 +383,7 @@ public boolean merge(ChannelHandlerContext ctx, Http2RemoteFlowController.FlowCo
         private final short weight;
         private final boolean exclusive;
 
-        private FlowControlledHeaders(Http2Stream stream, Http2Headers headers, int streamDependency, short weight,
+        public FlowControlledHeaders(Http2Stream stream, Http2Headers headers, int streamDependency, short weight,
```

Comment:
These doesn't have to be public; package-private should be enough and we save a few bytes.